CadRevealFbxProvider/FbxNodeToCadRevealNodeConverter.cs

```diff
@@ -131,7 +128,9 @@ IReadOnlySet<IntPtr> geometriesThatShouldBeInstanced
     )
     {
         var nodeGeometryPtr = FbxMeshWrapper.GetMeshGeometryPtr(node);
-        var transform = FbxNodeWrapper.GetTransform(node);
+        var worldTransform = node.WorldTransform;
+
+        var translation = node.GetLocalTransform().Translation;
```

Comment:
Is translation used?

Reply:
Woops. Forgot to remove it. Used it for some debugging